api/src/main/java/org/openmrs/CohortMembership.java

```diff
@@ -13,7 +13,11 @@
 import org.apache.commons.logging.LogFactory;
 
 import java.util.Date;
+import java.util.Objects;
```

Comment:
@vshl please organize your imports according to the https://wiki.openmrs.org/display/docs/Coding+Conventions